manifests/cf-manifest/spec/manifest/rds_broker_spec.rb

```diff
@@ -139,13 +139,20 @@
       it { expect(rds_properties["db_instance_class"]).to match(/^db\.[a-z0-9]+\.large$/) }
     end
 
-    shared_examples "large sized plans" do
+    shared_examples "old large sized plans" do
```

Comment:
Should we change this to be `deprecated large sized plans ` to make it linguistically consistent